### PR TITLE
Promotes Beepsky

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
After over a decade on the force, NT's brass has seen fit to finally officially recognize NT's longest serving and most dependable crewmember: Officer Beepsky! Beepsky has been formally granted the rank of Commander, as well as a commission over all of the bots on the station. As such, all other non-commissioned bots will automatically salute Beepsky upon crossing paths.

## Why It's Good For The Game
Recognizes a hero who is more dependable and heroic than any of us could ever hope to be

## Changelog
:cl: Ryll/Shaps
add: Officer Beepsky has been promoted to Commander Beepsky! He is now officially in command of all of the bots on the station, who will now show him proper respect by saluting him when they cross paths.
/:cl:

### CODE REVIEW

**Comment by pireamaineach** [comment]
Very good P.R., thank you.

**Comment by pireamaineach** [comment]
My only qualm is that you changed his actual name. He is officer beepsky, thanks.

**Comment by Ryll-Ryll** [comment]
It's always striking the first time you see a dear friend from the front lines wearing a white shirt, with a big fancy scrambled eggs peaked cap atop their neatly trimmed head. You remember the war stories you shared, the lives you've seen wasted mere inches from your own, and the shared friends you'll never see again. But it will pass. You think about all of the experiences your friend has accumulated over the course of their life, and you're happy for the fact that they've risen above the pack. Despite their new decorum and fancy outfit, you know that deep down, they're still one of you. One of the enlisted. They won't forget you either.

**Comment by ATH1909** [comment]
Should we modify the letter to Beepsky in his spawn room on... I think it was Boxstation?

**Comment by ATH1909** [comment]
You should also change:

```
/obj/effect/hallucination/simple/securitron
	name = "Securitron"
	desc = "The LAW is coming."
	image_icon = 'icons/mob/aibots.dmi'
	image_state = "secbot-c"
	var/victim

/obj/effect/hallucination/simple/securitron/New()
	name = pick ( "officer Beepsky", "officer Johnson", "officer Pingsky")
	START_PROCESSING(SSfastprocess,src)
	..()

```
to say "Commander Beep O'Sky" instead of "Officer Beepsky" (they misspelled his named smh).

**Comment by Ryll-Ryll** [comment]
I'll leave the letter as it was since that would be retconning his past, but I'll fix his name elsewhere

**Comment by wesoda25** [comment]
haha hEHE hEHE i sit in armory all day and can easily be killed if someone hides behind a weapon rack and uses a gun on me and then do nothing!!!

beepsky  is a fucking rat bastard and doesnt deserve a promotion. this pr is nonsense

**Comment by Ryll-Ryll** [comment]
That's Sgt. Armsky, soda, Beepsky is the one risking his life on the frontlines

**Comment by wesoda25** [comment]
oh fuck i was completely out of place. Beepsky is a national hero and this is an amazing PR

**Comment by PKPenguin321** [comment]
That's Commander Officer Beepsky to you

**Comment by Ryll-Ryll** [comment]
Commander Officer Beepsky works, I'll do that tomorrow

**Comment by pireamaineach** [comment]
I think Officer Beepsky is best. You can give him a little flavour in desc and have other borgs salute him but changing his name is too much of a shake up of a classic character.

**Comment by Tlaltecuhtli** [comment]
make it change name if you hit beepsky with a medal of captancy

**Comment by Ryll-Ryll** [comment]
Change is the whole point of it, it's honoring him in a little way that will be noticeable. Everything will be fine

**Comment by Tlaltecuhtli** [comment]
name doesnt change every year to spacestation2020 because its more iconic same for officer beepsky

**Comment by pireamaineach** [comment]
Okay so hear me out.

Default name is Officer Beepsky.

We make it so Beepsky can wear hats which rename him, like Ian.

**Comment by Ty-the-Smonk** [comment]
Give beepsky a knife

File: code/modules/mob/living/simple_animal/bot/secbot.dm [hunk 1]
```diff
@@ -44,15 +44,17 @@
 	var/payment_department = ACCOUNT_SEC
 
 /mob/living/simple_animal/bot/secbot/beepsky
-	name = "Officer Beep O'sky"
-	desc = "It's Officer Beep O'sky! Powered by a potato and a shot of whiskey."
+	name = "Commander Beep O'sky"
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/bot/secbot.dm]: beepsky is a much better name than beep o sky

Comment (Ryll-Ryll) [reply code/modules/mob/living/simple_animal/bot/secbot.dm]: Beepsky is the nickname for Beep O'Sky

**Comment by Ryll-Ryll** [comment]
@tlal what? It wasn't called ss13 because it was made in 2013 dude

Officer is a title, Beepsky/Beep O'Sky is the name and important part that people care about. He's still the Beepsky everyone knows and loves

**Comment by pireamaineach** [comment]
Bad

**Comment by Ryll-Ryll** [comment]
You're bad

**Comment by trollbreeder** [comment]
space station 13 was first made in 2003, back when the spacemen looked like they were made in ms paint
in 2006 goon started doing their own ss13
and in 2010 TG started doing their own ss13
this is literally on the tg station wiki, space station 13 wasn't named after the year 2013 else it'd be space station 3

File: code/modules/mob/living/simple_animal/bot/secbot.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/mob/living/simple_animal/bot/secbot.dm]: beep o sky is a lame name and literally no one uses it

**Comment by Fikou** [comment]
> space station 13 was first made in 2003, back when the spacemen looked like they were made in ms paint
> in 2006 goon started doing their own ss13
> and in 2010 TG started doing their own ss13
> this is literally on the tg station wiki, space station 13 wasn't named after the year 2013 else it'd be space station 3

more like trollfeeder

**Comment by cacogen** [comment]
it's called ss13 because it took 13 attempts to get byond to run it

**Pull request merged**